example/lambdas/snsS3Test/index.js

```diff
@@ -32,12 +32,12 @@ async function handleGranules(event) {
   const s3 = new S3();
   const messageString = event.Records[0].Sns.Message;
   const granuleRecord = JSON.parse(messageString);
-  if (!granuleRecord.granuleId) {
+  if (!granuleRecord.record.granuleId) {
```

Comment:
What should happen here if `granuleRecord.record` is not set?

Reply:
It will break. 
But record should be set in every case. If it's not then something is wrong and we should be notified about it.